src/Symfony/Component/Config/Loader/FileLoader.php

```diff
@@ -70,8 +74,14 @@ public function getLocator()
      * @throws FileLoaderImportCircularReferenceException
      * @throws FileLocatorFileNotFoundException
      */
-    public function import($resource, $type = null, $ignoreErrors = false, $sourceResource = null)
+    public function import($resource, $type = null, $ignoreErrors = self::IGNORE_NONE, $sourceResource = null)
     {
+        if (!\is_int($ignoreErrors)) {
+            @trigger_error(sprintf('Passing a boolean to argument "$ignoreErrors" of "%s()" is deprecated since Symfony 4.4, pass an integer instead.', __METHOD__), E_USER_DEPRECATED);
```

Comment:
I would say `pass one of the FileLoader::IGNORE_* constants instead`.

Reply:
Changed. Thank you.